common/lib/capa/capa/templates/jsinput.html

```diff
@@ -52,6 +52,6 @@
   % endif
 
   % if msg:
-      <span class="message">${msg|n}</span>
+      <span class="message" aria-describedby="iframe_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
@cptvitamin Is there anything in this template that makes sense to point to? This is "Custom Javascript Display and Grading" (the "cone and cube" problem).